hooks

```diff
@@ -0,0 +1 @@
+/home/katz/Documents/olin/hydro/hooks/hooks
```

Comment:
This actually should be code instead of a symlink, the path will be different for everyone. Need to see if there is a way to symlink and include the code here